src/shared/Footer.js

```diff
@@ -13,7 +13,6 @@ const Footer = () => {
         alignItems="center"
         spacing={2}
         width="100%"
-        style={{ margin: 20 }}
       >
         <Grid item lg={2} xs={12} sm={12} xl={2}>
           <center>
```

Comment:
Dont use center HTML

Reply:
i have removed the center tag from the code